Tests/PhishingDetectionTests/PhishingDetectionDataActivitiesTest.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  File.swift
+//  
+//
+//  Created by Thom on 04/05/2024.
+//
+
+import Foundation
+import XCTest
+@testable import BrowserServicesKit
+@testable import PhishingDetection
+
+class PhishingDetectionDataActivitiesTest: XCTestCase {
```

Comment:
If you can be bothered you can inject the intervals and set ito to like 1 second from the test anc check it’s called twice or something